desktop/public_desktop/index.html

```diff
@@ -8,48 +8,24 @@
-						xmlns="http://www.w3.org/2000/svg"
-						xmlns:xlink="http://www.w3.org/1999/xlink"
-						width="30px"
-						height="30px"
-						viewBox="0 0 35 35"
-						version="1.1"
-						data-src="wordpress.svg">
-						<desc>Created with Sketch.</desc>
-						<defs></defs>
-						<g stroke="none" stroke-width="1" fill="none" fill-rule="evenodd">
-							<g transform="translate(-842.000000, -1318.000000)" fill="#9598A8">
-								<g transform="translate(842.000000, 1318.000000)">
-									<path d="M26.8341102,8.52963741 C27.8868142,10.4504209 28.485865,12.6546949 28.485865,14.9994444 C28.485865,19.9748149 25.7895015,24.3186015 21.7801152,26.6562082 L25.8991847,14.7467474 C26.6685538,12.8227893 26.9247456,11.2843847 26.9247456,9.91645547 C26.9247456,9.41995016 26.892047,8.95915895 26.8341102,8.52963741 Z M16.8585019,8.67646203 C17.670411,8.63376384 18.4020021,8.54836747 18.4020021,8.54836747 C19.1286726,8.46233618 19.0429579,7.39440532 18.3159699,7.43694478 C18.3159699,7.43694478 16.1315139,7.60837244 14.7213477,7.60837244 C13.3962614,7.60837244 11.1695829,7.43694478 11.1695829,7.43694478 C10.4422775,7.39440532 10.3570389,8.50535182 11.0840269,8.54836747 C11.0840269,8.54836747 11.771967,8.63376384 12.4986376,8.67646203 L14.5996011,14.4334157 L11.6478394,23.2846387 L6.73701977,8.67646203 C7.54956375,8.63376384 8.28036127,8.54836747 8.28036127,8.54836747 C9.00655559,8.46233618 8.92084085,7.39440532 8.19417034,7.43694478 C8.19417034,7.43694478 6.01019053,7.60837244 4.59986561,7.60837244 C4.3468484,7.60837244 4.04859285,7.602182 3.73192451,7.59202332 C6.14336584,3.93109031 10.2881497,1.51348406 14.9999206,1.51348406 C18.5107329,1.51348406 21.7075752,2.85585791 24.1067942,5.05425897 C24.0486987,5.05076693 23.9920317,5.04346538 23.9321901,5.04346538 C22.6074212,5.04346538 21.6674162,6.19742753 21.6674162,7.43694478 C21.6674162,8.54836747 22.3086895,9.48867995 22.9921851,10.5999439 C23.505045,11.4981931 24.1040958,12.6519966 24.1040958,14.3192893 C24.1040958,15.4734102 23.6601252,16.8129269 23.0778999,18.6789488 L21.7323372,23.1733695 L16.8585019,8.67646203 Z M14.9999206,28.4865159 C13.6761041,28.4865159 12.3984783,28.29239 11.1899005,27.936995 L15.2367473,16.1791207 L19.3818486,27.5360448 C19.4091503,27.6027111 19.4426426,27.6641393 19.478357,27.7219168 C18.0766036,28.2154063 16.5704051,28.4865159 14.9999206,28.4865159 Z M1.51381753,14.9994444 C1.51381753,13.0442167 1.93318483,11.1881948 2.68160149,9.51106079 L9.11481013,27.1366818 C4.61557998,24.9509791 1.51381753,20.3375114 1.51381753,14.9994444 Z M14.9999206,0 C6.72892449,0 0,6.72869456 0,14.9994444 C0,23.2705118 6.72892449,30 14.9999206,30 C23.270758,30 30,23.2705118 30,14.9994444 C30,6.72869456 23.270758,0 14.9999206,0 Z" id="wordpress-logo-simplified-cmyk-15"></path>
-								</g>
-							</g>
-						</g>
-					</svg>
+          <svg xmlns="http://www.w3.org/2000/svg" viewBox="-2 -2 24 24" width="36" height="36" role="img" aria-hidden="true"
```

Comment:
Suggestion - maybe it's just me, but perhaps we can use dimensions `width="34" height="34"` here instead? Subtle but looks a little nicer with a more compact logo.

Reply:
That sounds good to me. I updated those sizes now.